systemtest/src/main/java/io/strimzi/systemtest/logs/TestExecutionWatcher.java

```diff
@@ -25,24 +26,30 @@
 
     @Override
     public void handleTestExecutionException(ExtensionContext extensionContext, Throwable throwable) throws Throwable {
-        String testClass = extensionContext.getRequiredTestClass().getName();
-        String testMethod = extensionContext.getRequiredTestMethod().getName();
-        collectLogs(testClass, testMethod);
+        if (!(throwable instanceof TestAbortedException)) {
```

Comment:
Just curious, is there some chance to use Junit5 method to do the same behaviour as you do here? AFAIK, `opentest4j` is used also by Junit5 this as their dependency.

Reply:
This is what junit5 uses.